content/en/docs/configuring-sdk/_index.md

```diff
@@ -67,19 +67,250 @@ export AWS_REGION=us-west-2
+
+If you have configured your instance to use {{% alias service=IAM %}} roles, the SDK uses
+these credentials for your application automatically. You don't need to
+manually specify these credentials.
+
+### Shared Credentials File
+
+A credential file is a plaintext file that contains your access keys.
+The file must be on the same machine on which you're running your
+application. The file must be named `credentials` and located in the
+`.aws/` folder in your home directory. The home directory can vary by
+operating system. In Windows, you can refer to your home directory by
+using the environment variable :code:`%UserProfile%`. In Unix-like systems, you
+can use the environment variable :code:`$HOME` or :code:`~` (tilde).
+
+If you already use this file for other SDKs and tools (like the {{% alias tools=CLI %}}),
+you don't need to change anything to use the files in this SDK. If
+you use different credentials for different tools or applications, you
+can use *profiles* to configure multiple access keys in the same
+configuration file.
+
+#### Creating the Credentials File
+
+If you don't have a shared credentials file (`.aws/credentials`), you
+can use any text editor to create one in your home directory. Add the
+following content to your credentials file, replacing
+`<YOUR_ACCESS_KEY_ID>` and `<YOUR_SECRET_ACCESS_KEY>` with your
+credentials.
+
+```ini
+[default]
+aws_access_key_id = <YOUR_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_SECRET_ACCESS_KEY>
+```
+
+The `[default]` heading defines credentials for the default profile,
+which the SDK will use unless you configure it to use another profile.
+
+You can also use temporary security credentials by adding the session
+tokens to your profile, as shown in the following example:
+
+```ini
+[temp]
+aws_access_key_id = <YOUR_TEMP_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_TEMP_SECRET_ACCESS_KEY>
+aws_session_token = <YOUR_SESSION_TOKEN>
+```
+
+#### Specifying Profiles
+
+You can include multiple access keys in the same configuration file by
+associating each set of access keys with a profile. For example, in your
+credentials file, you can declare multiple profiles, as follows.
+
+```ini
+[default]
+aws_access_key_id = <YOUR_DEFAULT_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_DEFAULT_SECRET_ACCESS_KEY>
+
+[test-account]
+aws_access_key_id = <YOUR_TEST_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_TEST_SECRET_ACCESS_KEY>
+
+[prod-account]
+; work profile
+aws_access_key_id = <YOUR_PROD_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_PROD_SECRET_ACCESS_KEY>
+```
+
+By default, the SDK checks the `AWS_PROFILE` environment variable to
+determine which profile to use. If no `AWS_PROFILE` variable is set,
+the SDK uses the default profile.
+
+If you have an application named `myapp` that uses the SDK, you can
+run it with the test credentials by setting the variable to
+`test-account myapp`, as shown in the following command.
+
+```bash
```

Comment:
Although this is a bash command, I wouldn't use bash syntax coloring here (or for any other bash commands). Bash syntax coloring is really for scripts. Using it for commands can cause shell keywords to be colored when they show up as part of string or filenames. That's my experience, anyway.